bootstrap/app.php

```diff
@@ -1,5 +1,18 @@
 <?php
 
+/*
+|--------------------------------------------------------------------------
+| Restart If Xdebug Is Loaded
+|--------------------------------------------------------------------------
+|
+| If the Xdebug extension is loaded, the application will automatically
+| restart unless the environment variable PINT_ALLOW_XDEBUG is set. This
+| ensures the application runs without the performance overhead of Xdebug.
+|
+*/
+
+(new Composer\XdebugHandler\XdebugHandler('PINT'))->check();
+
```

Comment:
If you want update the bootstrap file, probably you'll need to design a contract for it. 

Think about the idea of create a instance just to run one method and after that, release it to the void. 

If you read more forward or deep, you'll understand that they bind classes to contracts.